src/views/OnlineLeaguePlayerLanding/OnlineLeaguePlayerLanding.test.tsx

```diff
@@ -48,6 +48,9 @@ describe("COOL Player landing tests", () => {
     test("logged out player arrival", async () => {
         jest.spyOn(ogs_hooks, "useUser").mockReturnValue({ ...TEST_USER, anonymous: true });
 
+        location.pathname = "/online-league/league-player";
```

Comment:
I realize now, `pathname` isn't used in the code, so this line might be omitted.